### DISCUSSION TITLE
pandoc not taking the pdflatex binary from the PATH (within RStudio)

### DISCUSSION THREAD

**Discussion by JJ** [jgm/pandoc#10292] [Q&A] [answered]
My Ubuntu has an obsolete version of TeXLive installed; I have installed the new one, and set all possible paths, including RSTUDIO_PDFLATEX. `pandoc` seems to be still using `/usr/bin/pdflatex` and I can't seem to find a way to tell RStudio to tell `pandoc` to use another one. I've tried setting `latex_engine`, but that seems to work only for the general name of the binary, not the path.
I can symlink the new binary from the old, but I'm kind of wary of doing that. Uninstalling the old version might also break some builds I have. Is there any way to solve this?

**Comment by jgm** [comment, marked as answer]
Please address this question to rstudio developers, as it concerns rstudio-specific things.

On the pandoc command line, you certainly can specify a full path with `--pdf-engine`.

Pandoc itself won't pay any attention to `RSTUDIO_PDFLATEX`, but putting your new LaTeX in your `PATH` (ahead of he old one) should make a difference.
